### PR TITLE
feat: benchmark folder added in infino

### PR DESCRIPTION
**Pull request by savannahar68**
## What does this PR do?
- This PR adds benchmark package to parent package

## How does this PR work? (optional)
- Mentioned in README.md of benches folder

## Refer to a related PR or issue link
- https://github.com/infinohq/infino/issues/10

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,6 @@ Cargo.lock
 
 # the index created by tsldb if default config is used
 index
+
```
Comment (vinaykakade) [inline .gitignore:16]: Pl add a comment regarding what there are. e.g. "# target binary and index created by running benchmark"

File: benches/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# Benchmark - Elasticsearch and Prometheus Comparison with Infino
```
Comment (vinaykakade) [inline benches/README.md:1]: - Remove prometheus for now, and we haven't done that yet.
- Please only keep the latest results for infino, es, and tantivy (in that order). 
- Also, tsldb should be compared with lucene, and not es. Can you add that comparison?

**Review by vinaykakade — APPROVED** [review]
Suggestions above. The addition of lucene can be done incrementally later too.

**Comment by savannahar68** [comment]
@vinaykakade ready to merge.

**Pull request merged**